content/doc/book/pipeline/jenkinsfile.adoc

```
@@ -130,25 +103,18 @@ node {
 }
 // Declarative not yet implemented //
 ----
-<1> The `checkout` step will checkout code from source control; `scm` is a
-special variable which instructs the `checkout` step to clone the specific
-revision which triggered this Pipeline run.
+<1> The `checkout` step will checkout code from source control; `scm` is a special variable which instructs the `checkout` step to clone the specific revision which triggered this Pipeline run.
 
 
 
 === Build
 
-For many projects the beginning of "work" in the Pipeline would be the "build"
-stage. Typically this stage of the Pipeline will be where source code is
-assembled, compiled, or packaged. The `Jenkinsfile` is *not* a replacement for an
-existing build tool such as GNU/Make, Maven, Gradle, etc, but rather can be
-viewed as a glue layer to bind the multiple phases of a project's development
-lifecycle (build, test, deploy, etc) together.
+For many projects the beginning of "work" in the Pipeline would be the "build" stage.
+Typically this stage of the Pipeline will be where source code is assembled, compiled, or packaged.
+The `Jenkinsfile` is *not* a replacement for an existing build tool such as GNU/Make, Maven, Gradle, etc, but rather can be viewed as a glue layer to bind the multiple phases of a project's development lifecycle (build, test, deploy, etc) together.
```

```suggestion
The `Jenkinsfile` is *not* a replacement for an existing build tool such as GNU/Make, Maven, Gradle, or others, but rather can be viewed as a glue layer to bind the multiple phases of a project's development lifecycle (build, test, deploy) together.
```
Suggestion to remove `etc` as per the [microsoft style guide](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/e/etc)